src/plugins/orangehrmAdminPlugin/Service/EmploymentStatusService.php

```diff
@@ -58,18 +57,16 @@ public function setEmploymentStatusDao(EmploymentStatusDao $employmentStatusDao)
 
     /**
      * @param int $id
-     * @return EmploymentStatus
-     * @throws DaoException
+     * @return EmploymentStatus|null
      */
-    public function getEmploymentStatusById(int $id): EmploymentStatus
+    public function getEmploymentStatusById(int $id): ?EmploymentStatus
```

Comment:
a lot of the functions in this class are like wrapper functions. possible to remove?

Reply:
hm. I didn't remove after checking usage